source/layouts/layout.erb

```diff
@@ -65,10 +65,45 @@
     <footer class="group js-footer" id="footer" role="contentinfo">
       <div class="footer-wrapper">
         <div class="govuk-footer__navigation">
-          <div class="govuk-footer__section">
-            <ul class="govuk-footer__list govuk-footer__list--columns-1 footer-link-padding">
+          <div class="column-one-third">
+            <h2>About</h2>
+            <ul class="govuk-footer__list footer-link-padding">
               <li class="govuk-footer__list-item">
-                <a href="https://www.wifi.service.gov.uk/cookies" rel="cookie-policy" class="govuk-footer__link" >Cookies</a>
+                <%= link_to 'Our service', '/about-govwifi' %>
+              </li>
+              <li class="govuk-footer__list-item">
+                <%= link_to 'Terms and conditions', '/terms-and-conditions' %>
+              </li>
+              <li class="govuk-footer__list-item">
+                <%= link_to 'Performance', 'https://www.gov.uk/performance/govwifi' %>
+              </li>
+            </ul>
+          </div>
+          <div class="column-one-third">
+            <h2>Connect</h2>
+            <ul class="govuk-footer__list footer-link-padding">
+              <li class="govuk-footer__list-item">
+                <%= link_to 'Connect to GovWifi', '/about-govwifi/connect-to-govwifi' %>
+              </li>
+              <li class="govuk-footer__list-item">
+                <%= link_to 'Organisations using GovWifi', '/about-govwifi/organisations-using-govwifi' %>
+              </li>
+              <li class="govuk-footer__list-item">
+                <%= link_to 'Support', '/support' %>
+              </li>
+            </ul>
+          </div>
+          <div class="govuk-footer__section govuk-footer__list--columns-1">
```

Comment:
@Renny4Real this should also be a `column-one-third` if you are using that class on the other columns. Or does that not work?